packages/ui/src/lib/dropdownMenu/DropDownMenuItems.svelte

```diff
@@ -17,13 +21,20 @@ onMount(() => {
   } else {
     dropDownElement.style.top = '20px';
   }
+
+  if (window.innerWidth - clientX - NAV_BAR_WIDTH < dropDownWidth) {
```

Comment:
could you add some comments in the code to explain the purpose of the following changes ?

If I understand correctly it is to choose if the dropdown should be placed on the right or left of the user click position ?

Reply:
IMO dropdown menu should have no knowledge about navbar. It is independent UI component that can be used anywhere.

Reply:
@dgolovin the `NAV_BAR_WIDTH` has been removed